### DISCUSSION TITLE
Review of YouTube

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#241] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/youtube.svg">

**Comment by tjeerdvandijk** [comment]
When exporting to pdf, the link breaks

**Reply by Misokados** [reply]
Hello,
we're checking the situation. I will contact you shortly.

**Reply by Misokados** [reply]
Hello,
When saving to PDF, all OLE objects are converted to images because the PDF format does not support live embedded OLE objects. This is expected behavior and it's similar to how other editors (for example, MS Office) work.

**Comment by candideu** [comment]
I wish it actually embedded the YouTube videos into the slides Presentation. I need actual video playback.

**Reply by Misokados** [reply]
Hello,
Please describe your idea in details, we will check it out.

**Reply by candideu** [reply]
@Misokados 

If I insert a video using the tool, it simply adds a thumbnail image:

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/15d6a4ac-4690-4c85-9b1b-02a52db586f2" />

The video can't play back inside the presentation in Presentation mode. Clicking on it does nothing. I would like to be able to play the embedded video when I'm presenting, without leaving my presentation.

**Reply by Misokados** [reply]
Understood, we're checking the situation.

**Reply by Misokados** [reply]
Hello @candideu 
We have found a match with known bug (36175). We're working on it already. 
We will contact you once we have something to share.
